src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,33 @@
+import { getTransformTime } from '../../utils/transform.util';
+import './MovieDetails.css';
+
+export type MovieDetailsProps = {
+    imageUrl: string;
+    movieName: string;
+    releaseYear: number;
+    rating: number;
+    duration: number;
+    description: string;
+};
+
+const MovieDetails = ({ imageUrl, movieName, releaseYear, rating, duration, description }: MovieDetailsProps) => {
+    return (
+        <div className="movieDetails__container">
+            <img className="moviePoster" src={imageUrl} alt={movieName} />
+            <div className="movieDetails">
+                <div className="movieDetails__firstLine">
+                    <div className="movieName">{movieName}</div>
+                    <div className="movieRating">{rating}</div>
+                </div>
+                <div className="movieDetails__secondLine">
+                    <div className="movieYear">{releaseYear}</div>
+                    <div className="movieDuration">{getTransformTime(duration)}</div>
+                </div>
+
+                <div className="movieDescription">{description}</div>
+            </div>
+        </div>
+    );
+};
+
+export default MovieDetails;
```

Comment:
Like in regular function you can shorten code here and you don't have to use return, so is like:
const MovieDetails=()=>(<>jsx here</>);
it will 'add' return itself

Reply:
Not fully understand. Would like to discuss it :-)